libusb.go

```diff
@@ -169,13 +173,37 @@ type libusbIntf interface {
 }
 
 // libusbImpl is an implementation of libusbIntf using real CGo-wrapped libusb.
-type libusbImpl struct{}
+type libusbImpl struct {
+	logLevel         int
+	disableDiscovery bool
```

Comment:
Yes, good point. I wanted to keep the structure as is in the libusb options, but what you propose makes sense since gousb is supposed to be more 'go-like' rather than being just a libusb wrapper.